library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -65,6 +71,8 @@
     private CalendarDay currentMonth;
     private TitleFormatter titleFormatter = DEFAULT_TITLE_FORMATTER;
 
+    Set<DayViewDecorator> dayViewDecorators;
```

Comment:
Looking at RecyclerView, it's ItemDecorations are organized with an ArrayList. The overhead of a LinkedHashMap that backs a LinkedHashSet are a bit much for protecting against something I don't foresee happening often. I think if we add a `removeDecorator(decorator)` call, we can leave avoiding duplication to the people who use the library.